app/src/main/java/com/cornellappdev/volume/ui/components/general/PublicationComponents.kt

```diff
@@ -145,3 +156,53 @@ fun CreateHorizontalPublicationRow(
         }
     }
 }
+
+@Composable
+fun CreateFollowPublicationRow (
+    publication: Publication,
+    onPublicationClick: (Publication) -> Unit
+){
+    val title= publication.name
```

Comment:
nit: should be a space before the equal

Reply:
A lot of these small style issues can be fixed by android studio automatically! Always try and run this tool every so often to make sure things conform to style. At the top toolbar go to Code -> Reformat File